### DISCUSSION TITLE
Dynamically changing the vertical property

### DISCUSSION THREAD

**Discussion by ashBadCoder** [johnwalley/allotment#378] [Q&A]
Dynamically changing the vertical property does not work as expected, how to change the position more correctly, through a callback

_an example is attached:_
https://codesandbox.io/s/nested-forked-dwm95v
p.s.: if you hardcode, then everything works as it should
thanks in advance.

**Comment by johnwalley** [comment]
Hi @ashBadCoder. Thanks for the feedback and example. You can track progress on this here: https://github.com/johnwalley/allotment/issues/341.

VS Code does support changing the orientation so this should hopefully be relatively straightforward. I just need to find some time to do the implementation.
